src/main/java/com/github/technus/tectech/thing/metaTileEntity/single/GT_MetaTileEntity_DebugPowerGenerator.java

```diff
@@ -189,7 +221,7 @@ public long maxEUOutput() {
 
     @Override
     public long maxEUStore() {
-        return Math.abs((long)EUT*AMP)<<2;
+        return LASER ? Math.abs((long)EUT*AMP*24) : Math.abs((long)EUT*AMP)<<2 ;
```

Comment:
Why 24? where does this magic number come from?

Reply:
the normal laser hatch energy cap is eut * amps * 24 https://github.com/GTNewHorizons/TecTech/blob/master/src/main/java/com/github/technus/tectech/thing/metaTileEntity/hatch/GT_MetaTileEntity_Hatch_EnergyTunnel.java#L79